incubator/hnc/api/v1alpha1/hierarchy_types.go

```diff
@@ -46,6 +47,21 @@ const (
 	CannotUpdate      Code = "CannotUpdateObject"
 )
 
+// ClearConditionCriterion describes when a condition should be cleared: never, when it refers to an
+// object in a namespace that's no longer an ancestor, or when it refers to an object that's no
+// longer a descendant.
```

Comment:
A condition should *never* be cleared in these two cases? Shouldn't it be the opposite?

Reply:
Changed from "never" to "manual" and rewrote all the documentation - thanks for pointing that out.

Reply:
Nice doc, very clear!!